index.bs

```diff
@@ -253,6 +251,28 @@ This specification's [$client-hints-infrastructure/integration with Fetch$] is d
 
 
 
+Integration with HTML {#html-integration}
+-------
+
+This specification integrates with the [[!HTML]] specification in two places.
+
+First, it adds a new item to <a href="https://html.spec.whatwg.org/multipage/images.html#images-processing-model">images’ processing model</a>:
+
+An <{img}> element has a <dfn export>fallback sizes used state</dfn>, which is initially `false`.
+
+Second, it inserts two steps into the <a href="https://html.spec.whatwg.org/multipage/images.html#parsing-a-sizes-attribute">parsing a sizes attribute algorithm</a>.
+
+After step 2, it inserts the following step:
+
+<ol>
+<li>Set <var ignore=''>element</var>’s [=fallback sizes used state=] to `false`.
```

Comment:
_element_  can be a `source` element or a `link` element. I guess you can use _img_  instead (with a null check).